frontend/src/components/common/FileUpload.js

```diff
@@ -1,13 +1,26 @@
 import { map } from "lodash";
 import React from "react";
+import PropTypes from "prop-types";
 import { FilePond, File, registerPlugin } from "react-filepond";
 import "filepond/dist/filepond.min.css";
 import FilePondPluginFileValidateSize from "filepond-plugin-file-validate-size";
-registerPlugin(FilePondPluginFileValidateSize);
-
+import FilePondPluginFileValidateType from "filepond-plugin-file-validate-type";
 import { ENVIRONMENT } from "@/constants/environment";
 import { createRequestHeader } from "@/utils/RequestHeaders";
 
+registerPlugin(FilePondPluginFileValidateSize, FilePondPluginFileValidateType);
+
+const propTypes = {
+  uploadUrl: PropTypes.string.isRequired,
+  maxFileSize: PropTypes.string,
+  acceptedFileTypesMap: PropTypes.object,
```

Comment:
The way to avoid a linting error on an object with dynamic/inconsistent keys is to type-check the object values: `PropTypes.objectOf(PropTypes.string)`

Reply:
That works, thanks.